pkg/api/plugins.go

```diff
@@ -102,6 +107,16 @@ func (hs *HTTPServer) GetPluginList(c *models.ReqContext) response.Response {
 	return response.JSON(200, result)
 }
 
+func hidePluginInCatalog(cfg *setting.Cfg, pluginID string) bool {
```

Comment:
**Suggestion:** maybe a name like `shouldHidePluginInCatalog(...)` would express more what it is doing

```suggestion
func shouldHidePluginInCatalog(cfg *setting.Cfg, pluginID string) bool {
```